Rater.Domain/DataTransferObjects/MetricDto/MetricWinnerDto.cs

```diff
@@ -7,14 +7,14 @@
 
 namespace Rater.Domain.DataTransferObjects.MetricDto
 {
-    public class MetricResponseForResultDto
+    public class MetricWinnerDto
```

Comment:
TBH, I really didn't like this `Winner` keyword. It's not a competition it's should be something like best or etc.